test/lib/loaders/total.js

```diff
@@ -14,10 +14,10 @@ describe('total', () => {
 
     total.__Rewire__('gravity', gravity);
 
-    return total('foo/bar')
+    return total('foo/bar', { extra_option: 1 })
       .then(n => {
         gravity.args[0][0]
-          .should.equal('foo/bar?size=1&total_count=1');
+          .should.equal('foo/bar?extra_option=1&size=0&total_count=1');
```

Comment:
I could have sworn this worked previously. I guess that's the change you made above that I'm confused by